google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -61,13 +41,14 @@ public String toString() {
   private String environment;
 
   private String dockerFilePath;
-  private String appYamlPath;
+  private String yamlPath;
```

Comment:
Fair enough. Though, from what I could gather, app.yaml is really the only YAML usage in the entire plugin, so appYaml and yaml would mean the same thing.
I suggest that if we ever add utilities like blacklist, task queues, and any other configuration that uses yaml format, we rename this to mainYaml or so?
I'm quite happy with going away from appYaml terminology and would prefer to stay this way.